docs/samples/git-setup/clustertemplates/version_4.Y.Z/kustomization.yaml

```diff
@@ -15,6 +15,8 @@ resources:
 # sno-ran-du.v4-Y-Z-1 ClusterTemplate:
 - sno-ran-du/sno-ran-du-v4-Y-Z-1.yaml
 - sno-ran-du/clusterinstance-defaults-v1.yaml
+# sno-ran-du.v4-Y-Z-1-no-hwtemplate ClusterTemplate:
+- sno-ran-du/sno-ran-du-v4-Y-Z-1-no-hwtemplate.yaml
```

Comment:
nit: Would it be better to put this at the end of the list as it really is separate from the other example templates?

Reply:
+1